File: QuanLyPhongKham.Client/src/interfaces/responses/BacSiResponse.js
/**
 * @typedef {Object} BacSiResponse
 * @property {string} maBS - Doctor ID
 * @property {string} hoTen - Full name
 * @property {string} chuyenKhoa - Specialty
 * @property {string} soDienThoai - Phone number
 * @property {string} email - Email
 */

export const createBacSiResponse = (
	maBS = '',
	hoTen = '',
	chuyenKhoa = '',
	soDienThoai = '',
	email = ''
) => ({
	maBS,
	hoTen,
	chuyenKhoa,
	soDienThoai,
	email,
});

File: QuanLyPhongKham.Server/src/QuanLyPhongKham.DTOs/Responses/BacSiResponse.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace QuanLyPhongKham.DTOs.Responses
{
    public class BacSiResponse
    {
        public Guid MaBS { get; set; }
        public string HoTen { get; set; }
        public string? ChuyenKhoa { get; set; }
        public string? SoDienThoai { get; set; }
        public string? Email { get; set; }
    }
}
